pkg/flow/ent/schema/logtag.go

```diff
@@ -0,0 +1,28 @@
+package schema
```

Comment:
I'm beginning to think maybe instead of adding a new table we should just flatten it by adding a new column to the existing logmsg table.